migrations/cockroach/V1.98__keys.sql

```diff
@@ -0,0 +1,27 @@
+CREATE TABLE zitadel.projections.keys (
+    id STRING,
+    creation_date TIMESTAMPTZ NOT NULL,
+    change_date TIMESTAMPTZ NOT NULL,
+    resource_owner STRING NOT NULL,
+    sequence INT8 NOT NULL,
+    algorithm STRING DEFAULT '' NOT NULL,
+    use STRING DEFAULT '' NOT NULL,
+
+    PRIMARY KEY (id)
+);
+
+CREATE TABLE zitadel.projections.keys_private(
+    id STRING,
+    expiry TIMESTAMPTZ NOT NULL,
+    key JSONB NOT NULL,
+
+    PRIMARY KEY (id)
+);
+
+CREATE TABLE zitadel.projections.keys_public (
+    id STRING,
```

Comment:
if we query both keys via the keys projection i recommend a foreign key.
If deletes should not cascade to this table use the following:
```suggestion
    id STRING REFERENCES zitadel.projections.keys ON DELETE NO ACTION,
```